docs(readme): Add gimbal integration (#9083)
GoogleChrome/lighthouse: readme.md

## Related Projects
Other awesome open source projects that use Lighthouse.

> * **[Gimbal](https://labs.moduscreate.com/gimbal-web-performance-audit-budgeting)** - An [open source (MIT licensed)](https://github.com/ModusCreateOrg/gimbal) tool to meaure and analyze aspects of a web application and check them against thresholds. Users can produce Lighthouse reports along with checking built file sizes and unused source. Gimbal can also report these reports to a GitHub pull request.

Reviewer: ```suggestion
* **[Gimbal](https://labs.moduscreate.com/gimbal-web-performance-audit-budgeting)** - An [open source (MIT licensed)](https://github.com/ModusCreateOrg/gimbal) tool used to measure, analyze, and budget aspects of a web application. Gimbal also integrates reports with GitHub pull requests.
```
